### ISSUE TITLE
Batchnorm after convtranspose converted to wrong bias

### ISSUE THREAD

**Issue by Hyunseok-Kim0** [PINTO0309/onnx2tf#16]
### Issue Type

Others

### onnx2tf version number

1.1.23

### Download URL for ONNX

gist for reproduction : https://colab.research.google.com/gist/Hyunseok-Kim0/41920f44e994caa96271d604e5831775/onnx2tf_batchnorm_test.ipynb

### Parameter Replacement JSON

```json
N/A
```

### Description

First of all, thank you for sharing another wonderful tool. onnx2tf looks very stable and useful.

1. Purpose: Personal development

2. What : I tried to convert centernet (https://github.com/xingyizhou/CenterNet) to tflite. It was very impressive that initial conversion from onnx to tflite was completed without an error using onnx2tf. However, I found that generated tflite using onnx2tf shows different output comparing with the one generated from openvino2tensorflow. Most of network components were identical except the TransposeConv layer, which was followed by Batchnorm layer.

![Screenshot from 2022-11-16 16-24-20](https://user-images.githubusercontent.com/34959032/202114075-7624effd-9b7a-40a2-8462-6be931d53182.png)
part of tflite from openvino2tensorflow (generates correct output)

![Screenshot from 2022-11-16 16-24-39-1](https://user-images.githubusercontent.com/34959032/202114082-2947c494-674f-4b87-834a-a9af802b6dc9.png)
part of tflite from onnx2tf (generates wrong output)

As you see in the figures above, the TransposeConv layer from openvino2tensorflow has no bias while onnx2tf version has some bias. Also, the Add layers which is converted from Batchnorm layer has different values which are shown in the right side of images.

I think this problem is related with Batchnorm layer, but couldn't find solution.

3. How: please use the gist to reproduce this problem. (https://colab.research.google.com/gist/Hyunseok-Kim0/41920f44e994caa96271d604e5831775/onnx2tf_batchnorm_test.ipynb) 
The gist generates dummy pytorch network which contains only two layers, (Conv + Batchnorm) or (ConvTranspose + Batchnorm). After the conversion to tflite, the inference output is compared each other.
For the network with (Conv + Batchnorm), it shows same network structure and same outputs. However, (ConvTranspose + Batchnorm) shows different outputs.

![Screenshot from 2022-11-16 17-01-22](https://user-images.githubusercontent.com/34959032/202121870-1a9ded42-2d77-4503-9835-eba60e3b74c1.png)
target onnx

![Screenshot from 2022-11-16 17-01-03](https://user-images.githubusercontent.com/34959032/202121897-0088583b-4bf9-4dab-8602-e0134f872135.png)
tflite from openvino2tensorflow

![Screenshot from 2022-11-16 17-01-13](https://user-images.githubusercontent.com/34959032/202121946-65f3fd27-7d5a-44eb-9b99-8c9fb1064fc6.png)
tflite from onnx2tf

In tflite from onnx2tf, bias of Batchnrom layer is moved to Add layer, and another unknown bias value is added to TransposeConv layer. In tflite from openvino2tensorflow, TransposeConv layer has no bias and unknown value is added in Add layer.

Also, I found there is typo in ConvTranspose.py in line 68. `tf_layers_dict[input_weights.name]` instead of `tf_layers_dict[input_bias.name]`, but it does not look like cause of this problem..
https://github.com/PINTO0309/onnx2tf/blob/b2757407e33e5744884188b2e683f689eae8379a/onnx2tf/ops/ConvTranspose.py#L62-L70

I will keep try to find out the solution, please share your idea about this problem.

**Comment by PINTO0309** [comment]
Thanks for pointing that out.
It seems that I had introduced a bug in the `BatchNormalization` process by trying to force the model to be optimized. Therefore, it has been fixed.
![image](https://user-images.githubusercontent.com/33194443/202163255-616a4b24-4977-4265-82fe-4b57572bd8e1.png)

commit: https://github.com/PINTO0309/onnx2tf/commit/4cd0e2c7aeeb11f0690a44660490ce156236a8b1
release: https://github.com/PINTO0309/onnx2tf/releases/tag/1.1.24

>Also, I found there is typo in ConvTranspose.py in line 68. tf_layers_dict[input_weights.name] instead of tf_layers_dict[input_bias.name], but it does not look like cause of this problem..

You are certainly right. Although it was not directly related to this issue, we have determined the correction as you indicated. Thank you.

The problem should be fixed in v1.1.24.
